Dockerfile

```diff
@@ -20,6 +20,19 @@ COPY . .
 
 RUN make release-binary
 
+FROM alpine:3.13.1 AS gomplate
+
+ARG TARGETOS
+ARG TARGETARCH
+ARG TARGETVARIANT
+
+ENV GOMPLATE_VERSION=v3.9.0
+
+RUN wget -O /usr/local/bin/gomplate \
```

Comment:
Maybe use the slim version instead for a smaller binary: https://github.com/hairyhenderson/gomplate/releases

Reply:
Actually, does this work with every arm version? arm64, armv7? (The first one is GOARCH, the latter is variant AFAIK)

Reply:
Yes, it does! I have tested with armv7 and arm64.

Reply:
Awesome, thanks!